src/Sylius/Component/Core/spec/Order/Requirements/RequiredBillingAddressSpecificationSpec.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Component\Core\Order\Requirements;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\Address;
+use Sylius\Component\Core\Model\OrderInterface;
+
+class RequiredBillingAddressSpecificationSpec extends ObjectBehavior
+{
+    function it_is_not_satisfied_by_anonymous_object()
+    {
+        $this->isSatisfiedBy(new class(){})->shouldBe(false);
+    }
+
+    function it_is_satisfied_by_order(OrderInterface $order)
+    {
+        $order->getBillingAddress()->willReturn(new Address());
```

Comment:
can we make mock instead creating it by `new`/

Reply:
Yea, but what will it change?

Reply:
We do not use the `new` statement in specs (except for a few exceptions). If spec can do sth for us Imo he should :D
in this case, for me, mock will provide information about what an `address` is

Reply:
Alright, if we mock then let's mock